community/2017-events/12-contributor-summit/ContribSummitInformation.md

```diff
@@ -18,7 +18,7 @@ Times in CST
 01:00pm - Sessions resume  
 04:30pm - Happy Hour onsite  
 
-View the complete session schedule [here](/community/community/2017-events/12-contributor-summit/schedule.png).  
+View the complete session schedule [here](https://github.com/kubernetes/community/blob/master/community/2017-events/12-contributor-summit/schedule.png).  
```

Comment:
We should either use:
```
[here](/community/2017-events/12-contributor-summit/schedule.png)
```
or
```
[here](schedule.png)
```

These will ensure the links work on both master, branches, forks, etc.